docs/pages/tutorial/layout.md

```diff
@@ -0,0 +1,467 @@
+      <View style={styles.footerContainer}>
+        <Button label="Choose a photo" />
+        <Button label="Use this photo" />
+      </View>
+      /* @end */
+      <StatusBar style="auto" />
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  /* @hide // Styles that are unchanged from previous step are hidden for brevity. */
+  container: {    
+    flex: 1,
+    backgroundColor: "#25292e",
+    alignItems: "center",    
+  },
+  imageContainer: {
+    flex: 1,
+    paddingTop: 58,
+    backgroundColor: "transparent",
+  },
+  image: {    
+    width: 356,
+    height: 522,
+    borderRadius: 18,
+  },
+  /* @end */
+
+  footerContainer: {
+    flex: 1 / 3,
+    alignItems: "center",
+  },  
+});
+```
+
+The second button with the label "Use this photo" resembles the actual button from the layout. However, the first button needs more styling to match the layout design.
+
+## Step 7: Enhance the reusable button component
+
+To differentiate the two buttons, let's add another prop to the `Button` component called `isBorderLess` which is a boolean. When this prop's value is true, the unstyled button is displayed. Otherwise, the component with enhanced styling is displayed
+
+The styled button with the label "Choose a photo" also has an icon before the label. [`@expo/vector-icons` library](/guides/icons/#expovector-icons) provides icons from popular icon sets such as Ionicons, Feather, FontAwesome, MaterialCommunityIcons and so on. To load and display the icon on the button, let's use `FontAwesome` from the library.
+
+Modify the **Button.js** to add the following code snippet:
+
+<!-- prettier-ignore -->
+```js
+import { StyleSheet, View, Pressable, Text } from 'react-native';
+/* @info Import FontAwesome. */import FontAwesome from "@expo/vector-icons/FontAwesome";/* @end */
+
+
+export default function Button({ label, /* @info The prop isBorderLess has a default value of false. */ isBorderLess = false /* @end */ }) {
+  /* @info Conditionally render the borderless button when isBorderLess is true */
+  if (isBorderLess) {
+    return (
+      <View style={styles.buttonContainer}>
+        <Pressable style={styles.button} onPress={() => alert('You pressed a button.')}>
+          <Text style={styles.buttonLabel}>{label}</Text>
+        </Pressable>
+      </View>
+    );
+  }
+  /* @end */
+
+  return (
+    <View
+      style={[
+        styles.buttonContainer,
+        { borderWidth: 4, borderColor: '#ffd33d', borderRadius: 18 },
+      ]}>
+      <Pressable
+        style={[styles.button, { backgroundColor: '#fff' }]}
+        onPress={() => alert('You pressed a button.')}>
+         /* @info The FontAwesome icon is used on the Pressable component. */
+        <FontAwesome name="picture-o" size={18} color="#25292e" style={styles.buttonIcon} />
+        /* @end */
+        <Text style={[styles.buttonLabel, { color: '#25292e' }]}>{label}</Text>
+      </Pressable>
+    </View>
+  );
```

Comment:
This is new code that we should highlight. I think that whenever new code is added, it should be highlighted.